scripts/asm-installer/asm_vm

```diff
@@ -945,8 +945,11 @@ uses_environ_workload_identity() {
   fi
 
   if [[ "$(retry 2 kubectl get configmap "${CONFIGMAP}" -n istio-system \
-    -ojsonpath='{.data.mesh}' | grep -x -c "^trustDomain: ${PROJECT_ID}.hub.id.goog$" \
-    || true)" -eq 0 ]]; then
+    -ojsonpath='{.data.mesh}' | grep -x -c "^trustDomain: ${PROJECT_ID}.\(hub\|svc\).id.goog$" \
+    || true)" -eq 1 ]]; then
```

Comment:
I guess we can do `-eq 0` here and we can just `false` below, so we don't need the else below.

Reply:
Forgot to change it back. Done.